dev/docker/ranger/.env

```diff
@@ -0,0 +1,88 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BUILD_HOST_SRC=false
+SKIPTESTS=true
+GIT_URL=https://github.com/apache/ranger.git
+# BRANCH=master
+BRANCH=release-ranger-2.4.0
+BUILD_OPTS=-q
+
+# To build Trino plugins package, use following PROFILE
+PROFILE=ranger-jdk11,!all,!linux
+
+# Java version for RangerBase image.
+# This image gets used as base docker image for all images.
+# Valid values: 8, 11, 17
+RANGER_BASE_JAVA_VERSION=11
+
+# Java version to use to build Apache Ranger
+# Valid values: 8, 11, 17
+# Trino builds on jdk 11 and above
+RANGER_BUILD_JAVA_VERSION=11
+
+# Java version to use to run Ranger Admin server
+# Valid values: 8, 11, 17
+RANGER_ADMIN_JAVA_VERSION=11
+
+# base & third party images
+UBUNTU_VERSION=20.04
+MARIADB_VERSION=10.7.3
+POSTGRES_VERSION=12
+ENABLE_DB_MOUNT=true
+ZK_VERSION=3.5.9
+SOLR_VERSION=8
+
+# service versions
+HADOOP_VERSION=3.3.0
+HBASE_VERSION=2.4.6
+HIVE_VERSION=3.1.2
+HIVE_HADOOP_VERSION=3.1.1
+KAFKA_VERSION=2.8.1
+KNOX_VERSION=1.4.0
+TRINO_VERSION=377
+
+# versions of ranger services
+RANGER_VERSION=2.4.0
+KMS_VERSION=2.4.0
+USERSYNC_VERSION=2.4.0
+TAGSYNC_VERSION=2.4.0
+
+RANGER_VERSION=2.4.0
+KMS_VERSION=2.4.0
+USERSYNC_VERSION=2.4.0
+TAGSYNC_VERSION=2.4.0
+
+# plugin versions
+# HDFS_PLUGIN_VERSION=2.4.0
+# YARN_PLUGIN_VERSION=2.4.0
+# HIVE_PLUGIN_VERSION=2.4.0
+# HBASE_PLUGIN_VERSION=2.4.0
+# KAFKA_PLUGIN_VERSION=2.4.0
+# KNOX_PLUGIN_VERSION=2.4.0
+# TRINO_PLUGIN_VERSION=2.4.0
```

Comment:
@unknowntpo I think we can delete these comments?

Reply:
Ok, deleted.